src/cli/commands/append/appendManager.ts

```diff
@@ -230,4 +256,35 @@ export class AppendManager {
     await this.osmCommandRunner.mergeChanges([`${diffPath}`, `--output=${simplifiedDiffPath}`]);
     return simplifiedDiffPath;
   }
+
+  private async getRemoteResources(resouces: RemoteResource[]): Promise<void> {
+    this.logger.info({
+      msg: 'getting remote resources from bucket',
+      projectId: this.stateTracker.projectId,
+      count: resouces.length,
+      bucketName: this.s3Client.bucketName,
+    });
+
+    const getResourcePromises = resouces.map(async (resource) => {
+      const stream = await this.s3Client.getObjectWrapper(resource.key);
+      const content = await streamToString(stream);
+
+      if (resource.type === 'script') {
+        const localScriptPath = join(DATA_DIR, resource.key);
+        await createDirectory(getFileDirectory(localScriptPath));
+        await fsPromises.writeFile(localScriptPath, content);
+        return;
+      }
+
+      const geojson = JSON.parse(content) as Feature | Geometry;
+      const isValid = geojsonValidator(geojson);
+      if (!isValid) {
+        throw new InvalidGeojsonError(`geojson with key ${resource.key} is invalid`);
+      }
+
+      geometries[resource.key] = geojson;
+    });
+
+    await Promise.all(getResourcePromises);
+  }
```

Comment:
I think the logic of remote resources from S3 should not be included inside appendManager, also the assertion of the geojson should be in a separate module.

Moreover, the app should retrieve the geojson file on startup, so maybe it should be injected into the app while bootstrapping, @CptSchnitz wdyt?

Reply:
why would we share the same configuration for multiple services? in our use case a .geojson file should be fetched at startup describing the area for relevant tiles

Reply:
the fetching is now done in another module.